src/v/redpanda/CMakeLists.txt

```diff
@@ -35,6 +35,9 @@ v_cc_library(
     v::config
 )
 
+find_program(RPK_BINARY_PATH rpk REQUIRED)
+message(STATUS "XXX--> " ${RPK_BINARY_PATH})
+
```

Comment:
This won't work on systems that do not have `rpk` installed yet.
https://buildkite.com/redpanda/redpanda/builds/28432#0187ddc2-a304-46d7-ac1b-3c73a5016514/424-611
From CI
```
CMake Error at src/v/redpanda/CMakeLists.txt:38 (find_program):
  Could not find RPK_BINARY_PATH using the following names: rpk
```